recipe/build-pkg.bat

```diff
@@ -0,0 +1,7 @@
+if "%cuda_compiler_version%"=="None" (
+    python -m pip install %GPU_OPT% . -vv
```

Comment:
Where is `GPU_OPT` defined?

Reply:
it's an artefact from previous iterations, forgot to delete...